kratos/python_scripts/restart_utility.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, model_part, settings):
         default_settings = KratosMultiphysics.Parameters("""
         {
             "input_filename"                 : "",
+            "io_foldername"                  : "",
```

Comment:
i would just say  `foldername`

Reply:
Why not `input_directory` to be consistent with `input_filename`?

Reply:
Well, I would not only label it as `input_directory`, as it can serve as both input (when loading restart) as well as output (when saving). Also, would make it more precise/correct as part of the bigger makeover. Will only merge if the MPI version works (still need to test...).

Reply:
OK. Fine for me.